docs/core-concepts/api-calls-anatomy/index.md

```diff
@@ -47,62 +57,113 @@ Every request data has several required fields which you must provide them and t
 Request data for this call is like so:
 
 ```ts
-interface IAssetIndex {
-  residence_list: 1; // it must always be `1`
-  passthrough?: object;
-  req_id?: number;
+{
+  residence_list: 1; // Api Call Method Name
+  passthrough?: object; // Optional
+  req_id?: number; // Optional
 }
 ```
 
-The `residence_list` field is the `Identifier Field` for the call and is required here, there may other required fields which are related to type of the request you wanna send. if you want to know more about `Residence List` check [it](https://api.deriv.com/api-explorer#residence_list) out here.
+The `residence_list` field is the `method name` for the call and is required, there may be other required fields which are related to type of the request you wanna send. if you want to know more about `Residence List` and other API calls please check them out in [API Explorer](https://api.deriv.com/api-explorer#residence_list).
 
 ### Optional Fields
 
 Every Call has several `Optional` fields as well, `passthrough` and `req_id` are always part of the request data but you can choose to opt-out and not use them.
 
 #### Passthrough Field
 
-Whatever you pass to this field will be returned back to you on `response` object, this can be helpful when you wanna simulate some kinda stateful like flow for your `requests` and `responses`.
+Whatever you pass to this field will be returned back to you on `response` object, this can be helpful when you need to simulate stateful flow for your `requests` and `responses`.
```

Comment:
Text adjustment:

Whatever you pass to this field will be returned back to you inside a `response` object, this can be helpful when you need to simulate a stateful flow for your `requests` and `responses`.